styleguide/derek/global-components/ceiling/ceilingDark/_ceilingDark.scss

```diff
@@ -0,0 +1,69 @@
+.ceiling-menuInline-item {
+  border-left: 1px solid $white;
+}
+
+.navbar-phoneiconDark,
+.navbar-chaticonDark,
+.navbar-emailiconDark {
+  color: $white;
+
+  &:hover {
+    color: $white;
+  }
+
+  &::before {
+    color: $white;
+  }
+}
+
+.ceilingDark-brand {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/rs-logo.svg');
+  background-position: left 50%;
+  background-repeat: no-repeat;
+  background-size: 100% 79%;
+  float: left;
+  height: 35px;
+  margin: 5px 10px;
+  position: relative;
+  transition-duration: .3s;
+  width: 150px;
+}
+
+@media only screen and (max-width: $screen-xs-max) {
+  @include responsive-invisibility('.ceilingDark-pageTitle');
+
+  .navbar-phoneiconDark,
+  .navbar-chaticonDark,
+  .navbar-emailiconDark {
+    padding: 15px 0;
+  }
+}
+
+@media only screen and (max-width: $screen-sm-max) {
+  .basement-iconListItem {
+    font-size: 1rem;
+  }
+}
+
+@media only screen and (min-width: $screen-md-min) {
+  margin: 0 (-$grid-gutter-width / 2);
+}
```

Comment:
What is this applying a margin to? It's not inside a selector so it doesn't seem like it's even doing anything here.